src/lib/model/ContentType.ts

```diff
@@ -95,7 +96,23 @@ export class ContentType extends HalResource {
   /**
    * Resources and actions related to a ContentType
    */
-  public readonly related = {};
+  public readonly related = {
```

Comment:
Missing some comments - they are used in the docs `npm run doc`

Reply:
added in some supporting comments